api/src/main/java/io/druid/segment/loading/DataSegmentPusherUtil.java

```diff
@@ -54,14 +54,25 @@ public static String getStorageDir(DataSegment segment)
   public static String getHdfsStorageDir(DataSegment segment)
```

Comment:
In particular, `getHdfsStorageDir` appears to be used only by `JobHelper.makeSegmentOutputPath`, and _that_ appears to be unused.